va/va.h

```diff
@@ -2433,6 +2433,16 @@ VAStatus vaBufferSetNumElements (
 #define VA_CODED_BUF_STATUS_FRAME_SIZE_OVERFLOW         0x1000
 #define VA_CODED_BUF_STATUS_AIR_MB_OVER_THRESHOLD	0xff0000
 
+/**
+ * \brief The coded buffer segment status contains frame encoding passes number
+ *
+ * This is the mask to get the number of encoding passes from the coded
+ * buffer segment status. 
+ * NUMBER_PASS(bit24~bit28): the number for encoding passes executed for the coded frame.
```

Comment:
Are you sure this mask has 5 bits ? If so, the mask should be 0x1f000000

Reply:
What is the default value if the backend driver doesn't set the pass number in the coded buffer?